benchmarks/benchmarks/bench_random.py

```diff
@@ -173,3 +173,16 @@ def time_bounded(self, bitgen, args):
                 self.rg.randint(0, max + 1, nom_size, dtype=dt)
             else:
                 self.rg.integers(0, max + 1, nom_size, dtype=dt)
+
+class Choice(Benchmark):
+    params = [1e3, 1e6, 1e8]
+
+    def setup(self, v):
+        self.a = np.arange(v)
+        self.rng = np.random.Generator()
```

Comment:
This is failing, either needs to be `Generator(PCG64())` or `default_gen()`

Reply:
Wasn't `default_gen` renamed to `default_rng`?

Reply:
yes, thanks

Reply:
fixed, thanks.